test/integration/targets/vcenter_folder/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - prepare_vmware_tests
```

Comment:
How about renaming this to `setup_vmware` to match other setup

Reply:
Well, both exist. Do you know the difference between the prepare_*_tests and the setup_*?